src/OpenTelemetry.Exporter.ZPages/Implementation/ZPagesExporterEventSource.cs

```diff
@@ -16,10 +16,9 @@
 
 using System;
 using System.Diagnostics.Tracing;
-using System.Globalization;
-using System.Threading;
+using OpenTelemetry.Internal;
 
-namespace OpenTelemetry.Exporter.Prometheus.Implementation
+namespace OpenTelemetry.Exporter.ZPages.Implementation
```

Comment:
Nice catch 😉

Reply:
Probably my old mistake

Reply:
Maybe a dumb question, I was wondering why do we need `OpenTelemetry.Exporter.ZPages.Implementation` namespace or it can be `OpenTelemetry.Exporter.ZPages`? It looks to me that everything under `OpenTelemetry.Exporter.ZPages` is implementation (we don't expect a concrete exporter to expose interfaces)?

Reply:
I think the idea of that is the implementation would be all internal files and in root, we would have the exposed ones. Besides that, i don't see a problem with everything in the same namespace. Should i change here or should i change in another PR?

Reply:
@cijothomas raised this a little while back in the context of the instrumentation projects https://github.com/open-telemetry/opentelemetry-dotnet/pull/766#issuecomment-653834950.

Probably makes sense to take a pass over everything and git rid off the `Implementation` namespace across the board.

Reply:
Probably makes sense as a separate PR.

Reply:
Definitely not blocking this PR. It looks like we have internal classes not under `Implementation` and we also have public classes under `Implementation` which have already defeated the purpose (if that was the purpose).

Reply:
after the merge, i can take a look and remove the implementation PR.